src/zarr/testing/strategies.py

```diff
@@ -277,6 +278,8 @@ def arrays(
     if a.metadata.zarr_format == 3:
         assert a.fill_value is not None
     assert a.name is not None
+    assert a.path == normalize_path(array_path)
```

Comment:
shall we delete line 287 then.

Reply:
https://github.com/zarr-developers/zarr-python/pull/2850/commits/64ef4897c5eb573c6113388344c94332564d0158